io.github.liberodark.OpenDrive.appdata.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'd prefer to see it upstreamed.

Reply:
Let me file an issue in the main repository for this to be considered...and I if so I'll modify the manifest to use that when that gets resolved :+1: